app/graphql/types/add_ons/object.rb

```diff
@@ -25,6 +25,8 @@ class Object < Types::BaseObject
 
       field :taxes, [Types::Taxes::Object]
 
+      field :netsuite_mapping, Types::IntegrationMappings::Netsuite::Object, null: true
```

Comment:
I have few comments here:)
1.) can we please introduce integration_mapping instead of netsuite_mapping (I am doing it also in another branch)
2.) can you explain why one add_on will have only one integration_mapping? There can be many connections so I will say that it should be `has_many` relation